frontend/src/pages/pipelines/global/modelCustomization/SshKeyFileUpload.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  DropEvent,
+  FileUpload,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+} from '@patternfly/react-core';
+import React from 'react';
+import { ZodIssue } from 'zod';
+import { ZodErrorHelperText } from '~/components/ZodErrorFormHelperText';
+
+const MAX_SIZE = 1024 * 1024; // 1 MB as bytes
```

Comment:
please reuse or expose the max size.  we have the same pipeline file max size stuff in
`frontend/src/concepts/pipelines/content/import/PipelineFileUpload.tsx`